doc/src/docs/InputOutputReference/01b-InputOutputReference.md

```diff
@@ -17523,8 +17511,340 @@ This output is the average part-load ratio of the heat pump water heater’s com
+
+#### Field: Inlet Air Humidity Schedule Name
+
+This alpha field contains the name of a schedule used to define the humidity of the inlet air to the heat pump evaporator and fan section. Schedule values must be entered as relative humidity fraction from 0 to 1 (e.g., a schedule value of 0.5 means 50%RH). This field is only used when the Inlet Air Configuration defined above is specified as ‘Schedule’, otherwise leave this field blank.
+
+#### Field: Inlet Air Zone Name
+
+This alpha field contains the name of the zone from which the heat pump evaporator and fan section draws some or all of its inlet air. This field is only used when the Inlet Air Configuration defined above is specified as ‘ZoneAirOnly’ or ‘ZoneAndOutdoorAir’.
+
+#### Field: Tank Name
+
+This alpha field contains the name of the specific water heater tank (WaterHeater:Mixed object) used by this heat pump water heater. This must be a tank of type `WaterHeater:Stratified`.
+
+#### Field: Tank Use Side Inlet Node Name
+
+This alpha field contains the name of the use side inlet node of the water heater tank used by this heat pump water heater. This name must match the Use Side Inlet Node Name in the water heater tank object (Ref. `WaterHeater:Stratified`). This field is required if the water heater tank use side nodes are connected to a plant loop, otherwise leave this field blank. When used, the branch object should reflect that this node is part of a `WaterHeater:HeatPump:WrappedCondenser` object (see branch object example below).
+
+#### Field: Tank Use Side Outlet Node Name
+
+This alpha field contains the name of the use side outlet node of the water heater tank used by this heat pump water heater. This name must match the Use Side Outlet Node Name in the water heater tank object (Ref. `WaterHeater:Stratified`). This field is required if the water heater tank use side nodes are connected to a plant loop, otherwise leave this field blank. When used, the branch object should reflect that this node is part of a `WaterHeater:HeatPump:WrappedCondenser` object (see branch object example below).
+
+#### Field: DX Coil Name
+
+This alpha field contains the name of the specific DX coil (`Coil:WaterHeating:AirToWaterHeatPump:Wrapped`) used by this heat pump water heater.
+
+#### Field: Minimum Inlet Air Temperature for Compressor Operation
+
+This numeric field contains the minimum inlet air dry-bulb temperature entering the air coil (evaporator) and fan section, in degrees Celsius, below which the heat pump compressor does not operate. The minimum inlet air dry-bulb temperature should be greater than or equal to -5°C. If this field is left blank, the default value is 10°C.
```

Comment:
The -5C for minimum inlet air temperature for compressor operation is in the idd, but it also issues a warning [in the code here](https://github.com/NREL/EnergyPlus/blob/beef5746f3e6145bec0a155a8fcca1b5527b8f64/src/EnergyPlus/WaterThermalTanks.cc#L1547-L1552). I think it will still run, but the warning will show up in the .err file. Should that be removed then?